components/sensor/pulse_meter.rst

```diff
@@ -64,12 +64,14 @@ measure the total consumed energy in kWh.
         pin: 12
         unit_of_measurement: 'kW'
         name: 'Electricity Usage'
+        internal_filter: 100ms
+        accuracy_decimals: 5
```

Comment:
3 makes more sense for kW as 0.001 is 1 Watt, not much point being more accurate than that

```suggestion
        accuracy_decimals: 3
```